### PR TITLE
chore: update readme.md to include depoy pipeline command

### PR DESCRIPTION
**Pull request by EstebanRomero84**
### What does this PR do?
Update readme.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,3 +54,94 @@ sf plugins
...
+  -w, --wait=<minutes>                            Number of minutes to wait for command to complete and display results.
+  --async                                         Run the command asynchronously.
+  --concise                                       Show concise output of the deploy result.
+  --verbose                                       Show verbose output of the deploy result.
+
+GLOBAL FLAGS
+  --json  Format output as json
+
+DESCRIPTION
+Before you run this command, changes in the branch must be merged in the source control repository
```
Comment (ekapner) [inline README.md]:
```suggestion
Before you run this command, changes in the branch must be merged in the source control repository.
```

File: README.md [hunk 2]
```diff
@@ -54,3 +54,94 @@ sf plugins
...
+
+-p, --devops-center-project-name=<value> Name of the DevOps Center project.
+
+-t, --tests=<value>... Apex tests to run when --test-level is RunSpecifiedTests.
+
+  Separate multiple test names with commas, and enclose the entire flag value in double quotes if a test contains a space.
+
+-v, --bundle-version-name=<value> Version name of the bundle
+
+  You must indicate the bundle version if deploying to the environment that correspond to the first stage after the bundling stage.
```
Comment (ekapner) [inline README.md]:
```suggestion
  You must indicate the bundle version if deploying to the environment that corresponds to the first stage after the bundling stage.
```

**Review by ekapner — COMMENTED** [review]
A couple very minor comments.

**Review by npiccolo — APPROVED** [review]
Looks good !!
Thanks Esteban !

**Pull request merged**